tools/esp8266/html/serial.html

```diff
@@ -20,10 +20,19 @@ <h1>Serial Console</h1>
                 <hr>
                 <h3>handle next buttons with care - test / debug only!!</h3>
                 <br/>
+                <label for="iv">Choose ur Inverter:</label>
```

Comment:
@lumapu the option value list is working for up to four inverters.
Should we make this dynamic based on the number of inverters set in the defines.h or config.h ?

Reply:
When yes, we should edit to see the Name of the inverter. Not the number. :)